Flow.Launcher.Core/ExternalPlugins/UserPlugin.cs

```diff
@@ -12,5 +14,12 @@ public record UserPlugin
         public string UrlDownload { get; set; }
         public string UrlSourceCode { get; set; }
         public string IcoPath { get; set; }
+        public DateTime LatestReleaseDate { get; set; }
+        public DateTime DateAdded { get; set; }
+
+        /* Label Data for Plugin Store */
+        public bool LabelNew { get; set; }
+        public bool LabelInstalled { get; set; }
+        public bool LabelUpdated { get; set; }
```

Comment:
Let's move this to a viewmodel for plugin store.

Reply:
How to? I'm not good at organizing things like this.